### PR TITLE
[Detection Rules] Adding Documents for v8.12.13 Pre-Built Detection Rules

### PR DESCRIPTION
**Pull request by protectionsmachine**
Security Doc updates for prebuilt security rule integration package version v8.12.13. Please note these are meant to merge into 8.12 only and not backport.

### CODE REVIEW

**Review by terrancedejesus — APPROVED** [review]

File: docs/detections/prebuilt-rules/downloadable-packages/8-12-13/prebuilt-rule-8-12-13-machine-learning-detected-dga-activity-using-a-known-sunburst-dns-domain.asciidoc [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+- To set up and run Packetbeat, follow https://www.elastic.co/guide/en/beats/packetbeat/current/setting-up-and-running.html[this] guide.
+
+
+*The following steps should be executed to install assets associated with the DGA Detection integration:*
+
+- Go to the Kibana homepage. Under Management, click Integrations.
+- In the query bar, search for Domain Generation Algorithm Detection and select the integration to see more details about it.
+- Follow the instructions under the **Installation** section.
+- For this rule to work, complete the instructions through **Configure the ingest pipeline**.
+```
```
Comment (natasha-moore-elastic) [inline docs/detections/prebuilt-rules/downloadable-packages/8-12-13/prebuilt-rule-8-12-13-machine-learning-detected-dga-activity-using-a-known-sunburst-dns-domain.asciidoc:79]: extra backticks?

File: docs/detections/prebuilt-rules/downloadable-packages/8-12-13/prebuilt-rule-8-12-13-machine-learning-detected-a-dns-request-with-a-high-dga-probability-score.asciidoc [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+- To set up and run Packetbeat, follow https://www.elastic.co/guide/en/beats/packetbeat/current/setting-up-and-running.html[this] guide.
+
+
+*The following steps should be executed to install assets associated with the DGA Detection integration:*
+
+- Go to the Kibana homepage. Under Management, click Integrations.
+- In the query bar, search for Domain Generation Algorithm Detection and select the integration to see more details about it.
+- Follow the instructions under the **Installation** section.
+- For this rule to work, complete the instructions through **Configure the ingest pipeline**.
+```
```
Comment (natasha-moore-elastic) [inline docs/detections/prebuilt-rules/downloadable-packages/8-12-13/prebuilt-rule-8-12-13-machine-learning-detected-a-dns-request-with-a-high-dga-probability-score.asciidoc:79]: extra backticks?

File: docs/detections/prebuilt-rules/downloadable-packages/8-12-13/prebuilt-rule-8-12-13-machine-learning-detected-a-suspicious-windows-event-predicted-to-be-malicious-activity.asciidoc [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+- To set up and run Winlogbeat, follow https://www.elastic.co/guide/en/beats/winlogbeat/current/winlogbeat-installation-configuration.html[this] guide.
+
+
+*The following steps should be executed to install assets associated with the LotL Attack Detection integration:*
+
+- Go to the Kibana homepage. Under Management, click Integrations.
+- In the query bar, search for Living off the Land Attack Detection and select the integration to see more details about it.
+- Follow the instructions under the **Installation** section.
+- For this rule to work, complete the instructions through **Configure the ingest pipeline**.
+```
```
Comment (natasha-moore-elastic) [inline docs/detections/prebuilt-rules/downloadable-packages/8-12-13/prebuilt-rule-8-12-13-machine-learning-detected-a-suspicious-windows-event-predicted-to-be-malicious-activity.asciidoc:78]: extra backticks?

File: docs/detections/prebuilt-rules/downloadable-packages/8-12-13/prebuilt-rule-8-12-13-machine-learning-detected-a-suspicious-windows-event-with-a-high-malicious-probability-score.asciidoc [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+- To set up and run Winlogbeat, follow https://www.elastic.co/guide/en/beats/winlogbeat/current/winlogbeat-installation-configuration.html[this] guide.
+
+
+*The following steps should be executed to install assets associated with the LotL Attack Detection integration:*
+
+- Go to the Kibana homepage. Under Management, click Integrations.
+- In the query bar, search for Living off the Land Attack Detection and select the integration to see more details about it.
+- Follow the instructions under the **Installation** section.
+- For this rule to work, complete the instructions through **Configure the ingest pipeline**.
+```
```
Comment (natasha-moore-elastic) [inline docs/detections/prebuilt-rules/downloadable-packages/8-12-13/prebuilt-rule-8-12-13-machine-learning-detected-a-suspicious-windows-event-with-a-high-malicious-probability-score.asciidoc:78]: extra backticks?

File: docs/detections/prebuilt-rules/downloadable-packages/8-12-13/prebuilt-rule-8-12-13-machine-learning-detected-a-dns-request-predicted-to-be-a-dga-domain.asciidoc [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+- To set up and run Packetbeat, follow https://www.elastic.co/guide/en/beats/packetbeat/current/setting-up-and-running.html[this] guide.
+
+
+*The following steps should be executed to install assets associated with the DGA Detection integration:*
+
+- Go to the Kibana homepage. Under Management, click Integrations.
+- In the query bar, search for Domain Generation Algorithm Detection and select the integration to see more details about it.
+- Follow the instructions under the **Installation** section.
+- For this rule to work, complete the instructions through **Configure the ingest pipeline**.
+```
```
Comment (natasha-moore-elastic) [inline docs/detections/prebuilt-rules/downloadable-packages/8-12-13/prebuilt-rule-8-12-13-machine-learning-detected-a-dns-request-predicted-to-be-a-dga-domain.asciidoc:79]: extra backticks?

File: docs/detections/prebuilt-rules/rule-details/machine-learning-detected-a-dns-request-predicted-to-be-a-dga-domain.asciidoc [hunk 6]
```diff
@@ -74,36 +74,8 @@ The DGA Detection integration consists of an ML-based framework to detect DGA ac
...
-        "malicious_probability": {
-          "type": "float"
-        }
-      }
-    }
-  }
-}
+- Follow the instructions under the **Installation** section.
+- For this rule to work, complete the instructions through **Configure the ingest pipeline**.
 ```
```
Comment (natasha-moore-elastic) [inline docs/detections/prebuilt-rules/rule-details/machine-learning-detected-a-dns-request-predicted-to-be-a-dga-domain.asciidoc:79]: extra backticks?

File: docs/detections/prebuilt-rules/rule-details/machine-learning-detected-a-dns-request-with-a-high-dga-probability-score.asciidoc [hunk 7]
```diff
@@ -74,36 +74,8 @@ The DGA Detection integration consists of an ML-based framework to detect DGA ac
...
-        "malicious_probability": {
-          "type": "float"
-        }
-      }
-    }
-  }
-}
+- Follow the instructions under the **Installation** section.
+- For this rule to work, complete the instructions through **Configure the ingest pipeline**.
 ```
```
Comment (natasha-moore-elastic) [inline docs/detections/prebuilt-rules/rule-details/machine-learning-detected-a-dns-request-with-a-high-dga-probability-score.asciidoc:79]: extra backticks?

File: docs/detections/prebuilt-rules/rule-details/machine-learning-detected-a-suspicious-windows-event-predicted-to-be-malicious-activity.asciidoc [hunk 8]
```diff
@@ -73,39 +73,8 @@ The LotL Attack Detection integration detects living-off-the-land activity in Wi
...
-      }
-    },
-    "blocklist_label": {
-      "type": "long"
-    }
-  }
-}
+- Follow the instructions under the **Installation** section.
+- For this rule to work, complete the instructions through **Configure the ingest pipeline**.
 ```
```
Comment (natasha-moore-elastic) [inline docs/detections/prebuilt-rules/rule-details/machine-learning-detected-a-suspicious-windows-event-predicted-to-be-malicious-activity.asciidoc:78]: extra backticks?

File: docs/detections/prebuilt-rules/rule-details/machine-learning-detected-a-suspicious-windows-event-with-a-high-malicious-probability-score.asciidoc [hunk 9]
```diff
@@ -73,39 +73,8 @@ The LotL Attack Detection integration detects living-off-the-land activity in Wi
...
-      }
-    },
-    "blocklist_label": {
-      "type": "long"
-    }
-  }
-}
+- Follow the instructions under the **Installation** section.
+- For this rule to work, complete the instructions through **Configure the ingest pipeline**.
 ```
```
Comment (natasha-moore-elastic) [inline docs/detections/prebuilt-rules/rule-details/machine-learning-detected-a-suspicious-windows-event-with-a-high-malicious-probability-score.asciidoc:78]: extra backticks?

**Pull request merged**